hmda/src/main/scala/hmda/validation/rules/lar/quality/_2025/Q621.scala

```diff
@@ -0,0 +1,26 @@
+package hmda.validation.rules.lar.quality._2025
+
+import hmda.model.filing.lar.LoanApplicationRegister
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateHmda._
+import hmda.validation.dsl.PredicateSyntax._
+import hmda.validation.dsl.{ValidationFailure, ValidationResult, ValidationSuccess}
+import hmda.validation.rules.EditCheck
+
+
+object Q621 extends EditCheck[LoanApplicationRegister] {
+  override def name: String = "Q621"
+
+  override def parent: String = "Q621"
+  // valid number between 4 and 12 digits
+  val regex1 = "^\\d{4,12}$"
```

Comment:
I think it would fit in our established code better to use length is greater or equal to 4 and less than or equal to 4

Reply:
```
val NMLSRLen = lar.larIdentifier.NMLSRIdentifier.length
NMLSRLen is lessThanOrEqualTo(12) and NMLSRLen is greaterThanOrEqualTo(3)`

Reply:
updated to reflect this suggestion